tests/golos.posting_test_api.hpp

```diff
@@ -174,6 +174,12 @@ struct golos_posting_api: base_contract_api {
         return push(N(setparams), _code, args()
             ("params", json_str_to_obj(json_params)));
     }
+    action_result closemssgs(account_name signer) {
```

Comment:
You can use default value for `signer`.

Reply:
@s-medvedev cannot. It willn't compile with `= _code`.